01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env node
+
+for (let i=1; i < 21; i++) {
```

Comment:
```suggestion
for (let i=1; i <= 20; i++) {
```

1 ~ 20 までの場合、こんな感じで `<=` を使った方が直感的かなと思いましたが。
このままでも問題ないので 🙆‍♂️ です！